.circleci/config.yml

```diff
@@ -153,6 +153,14 @@ jobs:
       - run:
           command: |
             ./.ci/validation.sh no-warning-imports-java-design-patterns
+  git-diff:
```

Comment:
please remove complately from circleci execution.
we need diff command only after "mvn ..." commands

Reply:
Changes Done

Reply:
Not done.


Please reply in comments only after you see it is done.
Please do not forget to push

Reply:
@romani please check again, I can see all the changes which I pushed.

Reply:
This file should not be changed at all

Reply:
I have done revert for all the changes.